fc/jira/triage_task.py

```diff
@@ -114,11 +115,10 @@ def create(self):
     def type_str(self) -> str:
         return 'Triage'
 
-    def score(self) -> tuple:
+    def score(self) -> Tuple:
         score = self._calculate_score()
-        self._update_triage_vfr(score)\
-            if self.score_value is None or self.score_value != score\
-            else cli_library.echo('not updating score for {} as new score is same'.format(self.id))
+        self._update_triage_vfr(score) if self.score_value is None or self.score_value != score else None
```

Comment:
This is just a glorified if statement and doesn't involve any conditional assignment.  So, I would recommend to change this to a simple if statement.

```
if self.score_value is None or self.score_value != score:
    self._update_triage_vfr(score)
```